content/en/blog/2023/opentelemetry-php-beta-release/index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: {{ Opentelemetry-php Beta Release }}
+linkTitle: PHP Beta Release # Mandatory, make sure that your short title. 
+date: 2023-01-25
+author: " [Bob Strecansky](https://github.com/bobstrecansky/) (Intuit)"
+draft: true
+
+OpenTelemetry PHP Beta Release Blog Post
+
+The OpenTelemetry PHP SIG is very excited to announce the release of [v1.0.0beta1](https://github.com/open-telemetry/opentelemetry-php/releases/tag/1.0.0beta1) of OpenTelemetry PHP.  This is the culmination of over 3 years of work done by the OpenTelemetry PHP team.  
+
+We are actively soliciting feedback from the development community for this library.  Please check out the repo, instrument your PHP app with the product, and open an issue if you’d like to see a bug squashed or a new feature added.
+
+We have lots of [examples](https://github.com/open-telemetry/opentelemetry-php/tree/main/examples) in the repo to get you started, or you can check out the [quote service](https://github.com/open-telemetry/opentelemetry-demo/blob/main/docs/services/quoteservice.md), a demo application built in PHP to showcase the library.
```

Comment:
```suggestion
We have lots of [examples](/docs/instrumentation/php/examples/) in the repo to get you started, or you can check out the [quote service](https://github.com/open-telemetry/opentelemetry-demo/blob/main/docs/services/quoteservice.md), a demo application built in PHP to showcase the library.
```

Pointing to the examples link here in the doc. Also do you want to call out the Getting Started in the docs here?